Source/santad/SNTApplicationCoreMetrics.mm

```diff
@@ -56,7 +79,7 @@ static void RegisterMemoryAndCPUMetrics(SNTMetricSet *metricSet) {
                        fieldNames:@[]
                          helpText:@"The virtual memory size of this process"];
   SNTMetricInt64Gauge *rsize =
-    [metricSet int64GaugeWithName:@"/proc/memory/resident_size"
+    [metricSet int64GaugeWithName:@ "/proc/memory/resident_size"
```

Comment:
Remove space after @. Surprised the linter didn't catch this.

Reply:
```suggestion
    [metricSet int64GaugeWithName:@"/proc/memory/resident_size"
```